cfgov/alerts/github_alert.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import github3
+
+
+class GithubAlert:
+    def __init__(self, credentials):
+        token = credentials.get(
+            'token',
+            os.environ.get('GITHUB_TOKEN')
```

Comment:
Minor suggestion: worth changing these things to something like this:

```py
token = credentials.get('token') or os.environ['GITHUB_TOKEN']
```

It'd be nice to have these things raise an exception if not defined, if you're falling back to them.